e2e/test/Common/CodeAnalysisOverrides.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics.CodeAnalysis;
+
+// Required by Configuration.*
+[assembly: SuppressMessage("Microsoft.Design", "CA1724:TypeNamesShouldNotMatchNamespaces")]
```

Comment:
do you know which classes are using this?

Reply:
I have not checked. I just copied everything the project was referring to.